### PR TITLE
remove old code from tests

### PR DESCRIPTION
**Pull request by peter-mouland**

### CODE REVIEW

File: src/shared/components/LogoSVG.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ export default class LogoSVG extends React.Component {
     }
     styles = assign({}, optStyles)
     return (
-      <Link to="/" style={styles.link}>
+      <Link to="/" style={styles.link} title="Lantern Homepage">
```
Comment (peter-mouland) [inline src/shared/components/LogoSVG.js]: tis good to do this... was meant to be another commit. oh well

File: src/shared/components/NavBar.js [hunk 2]
```diff
@@ -97,7 +97,7 @@ export default class NavBar extends React.Component {
           trigger="click"
           placement="bottom"
           overlay={
-            <Popover title="What is alpha?">
+            <Popover title="What is alpha?" id="alpha-description">
```
Comment (peter-mouland) [inline src/shared/components/NavBar.js]: stops React from moaning

**Comment by radiodario** [comment]
:+1:
